key/aziot-keyd/src/keys.rs

```diff
@@ -798,26 +798,26 @@ impl std::fmt::Display for DecryptError {
 impl std::error::Error for DecryptError {
 }
 
-fn keys_fn(f: impl FnOnce() -> sys::KEYGEN_ERROR) -> Result<(), KeysRawError> {
-	match f() {
-		sys::KEYGEN_SUCCESS => Ok(()),
+fn keys_ok(result: sys::AZIOT_KEYS_ERROR) -> Result<(), KeysRawError> {
+	match result {
+		sys::AZIOT_KEYS_SUCCESS => Ok(()),
```

Comment:
The naming on the result type is misleading. It should AZIOT_KEYS_STATUS since one of the return values is AZIOT_KEYS_SUCCESS and the rest are AZIOT_KEYS_ERROR.

Reply:
Good point. Fixed.